Formula/d/distill-cli.rb

```diff
@@ -0,0 +1,30 @@
+class DistillCli < Formula
+  desc "Use AWS Transcribe and Bedrock to create summaries of your audio recordings"
+  homepage "https://www.allthingsdistributed.com/2024/06/introducing-distill-cli.html"
+  url "https://github.com/awslabs/distill-cli.git",
+      using:    :git,
+      revision: "294c94ba552f4a5e40478567adffca4de0beaf84",
+      tag:      "v0.1.0-beta"
```

Comment:
we dont like to ship beta releases

Reply:
It's not beta, I will have them fix the tag and resubmit.

Reply:
sounds good.